Tools/TestWebKitAPI/Configurations/TestWebKitAPIBase.xcconfig

```diff
@@ -24,14 +24,15 @@
 
 GCC_ENABLE_OBJC_EXCEPTIONS = YES;
 
-GCC_PREPROCESSOR_DEFINITIONS = $(inherited) GTEST_API_=
+GCC_PREPROCESSOR_DEFINITIONS = $(inherited) GTEST_API_= $(GCC_PREPROCESSOR_DEFINITIONS_$(ENABLE_WEBGPU_SWIFT))
+GCC_PREPROCESSOR_DEFINITIONS_ENABLE_WEBGPU_SWIFT = ENABLE_WEBGPU_SWIFT;
 
 FRAMEWORK_SEARCH_PATHS = $(FRAMEWORK_SEARCH_PATHS_$(WK_COCOA_TOUCH));
 FRAMEWORK_SEARCH_PATHS_ = $(inherited) $(SYSTEM_LIBRARY_DIR)/PrivateFrameworks $(SYSTEM_LIBRARY_DIR)/Frameworks/WebKit.framework/Versions/A/Frameworks;
 FRAMEWORK_SEARCH_PATHS_cocoatouch = $(inherited);
 
 PROJECT_HEADER_SEARCH_PATHS = $(SRCROOT)/../../Source/WebKit/Platform $(SRCROOT)/../../Source/WebKit/Platform/IPC $(SRCROOT)/../../Source/WebKit/Platform/IPC/darwin $(SRCROOT)/../../Source/WebKit/Platform/IPC/cocoa $(SRCROOT)/../../Source/WebKit/Shared $(SRCROOT)/../../Source/WebKit/Shared/Cocoa $(SRCROOT)/../../Source/WebKit/Shared/cf $(SRCROOT)/../../Source/WebKit/Shared/API/C $(BUILT_PRODUCTS_DIR)/DerivedSources/WebKit $(inherited);
-
+HEADER_SEARCH_PATHS = $(inherited) $(OBJROOT)/WebGPU.build/WebGPU.build/DerivedSources $(OBJROOT)/WebGPU.build/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)/WebGPU.build/DerivedSource
```

Comment:
Could you remind me if we have precedent for reaching into other projects' intermediate build dir like this? It's pretty gross, and I worry that you are making assumptions about build directory structure that won't always be true (e.g. for a person who uses the Xcode-default derived data dir).

Reply:
There is a precedent in PAL<=>WebCore. and yes, It is gross. 
Although, I think for this PR this is not even needed anymore. Sorry for that. It's just a remnant of how I was thinking of doing this earlier which I should have removed.